code-review-chat/action.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Code Review Chat'
+description: 'Notify in chat about build results.'
+author: 'Jackson Kearl'
+inputs:
+  token:
+    description: GitHub token.
+    default: ${{ github.token }}
+    required: true
+  slack_token:
+    description: Slack token.
+    required: true
+  storage_connection_string:
+    description: Storage connection string.
+    required: false
+  notification_channel:
+    description: 'Which channel to send notifications to. Defaults to none.'
+    required: false
+runs:
+  using: 'node12'
```

Comment:
We probably want node16 don't we since that's what we're developing with?

Reply:
all the actions are on 12 at the moment. it just reached EOL end of last month, we should bump them all to 16 I guess